desc/compute/_basis_vectors.py

```diff
@@ -2444,7 +2467,13 @@ def _e_sub_theta_over_sqrt_g(params, transforms, profiles, data, **kwargs):
     profiles=[],
     coordinates="rtz",
     data=["e_theta", "theta_PEST_t", "e_zeta", "theta_PEST_z", "phi_t", "phi_z"],
-    aliases=["e_vartheta"],
+    aliases=[
```

Comment:
Change https://github.com/PlasmaControl/DESC/blob/d28daece96b8a45b2ff6e8d30c1ebc462001073c/desc/compute/data_index.py#L165
to 
```python
 and ({"r", "t", "z"}.issuperset(splits[-1]) or {"r", "v", "p"}.issuperset(splits[-1]))
```
and remove the permuations

Reply:
Yeah because a lot the aliases have | to specify which coordinate is constant. Also for e_theta_PEST, we should also add PEST|r,p because there is a "PEST|r, p" after e_theta_.

I can look at it right now but doesn't seem straightforward.